src/views/Accessibility/TestingTemplates/index.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from '@trussworks/react-uswds';
+
+import BreadcrumbNav from 'components/BreadcrumbNav';
+import PageHeading from 'components/PageHeading';
+
+import './index.scss';
+
+const TestingTemplates = () => {
+  const { t } = useTranslation('accessibility');
+  const vpatConformanceLevels: { name: string; description: string }[] = t(
+    'testingTemplates.vpatSection.subSection.item1.levels',
+    {
+      returnObjects: true
+    }
+  );
+  const testPlanList: string[] = t(
+    'testingTemplates.testPlanSection.itemsToProvide',
+    {
+      returnObjects: true
+    }
+  );
+
+  const remediationPlanList: string[] = t(
+    'testingTemplates.remediationPlanSection.itemsToProvide',
+    {
+      returnObjects: true
+    }
+  );
+
+  const tableOfContents = (
+    <div className=".accessibility-testing-templates__tableOfContents">
+      <p>Page contents</p>
+      <ul className="accessibility-testing-templates__table-of-contents">
+        <li>
+          <Link
+            href={`#testingTemplates_${t(
```

Comment:
These `href`'s don't need to come from i18n. URLs will always be static.

Because these affect the URL, they should probably be more human readable like:
```
href="vpat"
href="test-plan"
href="remediation-plan"
```

Reply:
I was thinking I needed to ensure that the ids I used were unique for the entire app.  If that's not necessary, then this def is simpler.

Reply:
The id's are scoped to the page. As long as they're unique on the page, it should be all good!

Reply:
Oh, good to know!  Thanks!